packages/components/Avatar/src/AvatarTokens.win32.ts

```diff
@@ -16,7 +16,7 @@ export const defaultAvatarTokens: TokenSettings<AvatarTokens, Theme> = (t: Theme
     iconSize: 16,
     iconColor: t.colors.neutralForeground3,
     ringColor: t.colors.transparentStroke,
-    borderColor: globalTokens.color.white,
+    borderColor: t.colors.neutralForeground3,
     borderWidth: t.name === 'HighContrast' ? 1 : 0,
```

Comment:
Nit: line 20 feels like a key each separate theme should be setting. Feels weird we checked for a specific theme's name.

Reply:
Yeah, but how would you do this in a different way?

Reply:
This is a component-specific setting. To some extent there will always be some HC-specific styling on a component, but it's hard to predict those on a Theme, so I think this is fine. Unfortunately with win32 we rely on the name to tell us what the current theme is. We could use the component property on the Theme but I think it'd be nice to avoid it since that information would end up living somewhere separate from the rest of the component information.

One way around having to change the borderWidth on HC is to keep the borderWidth and use transparentStroke (which is transparent on non-HC themes but uses a system brush color on HC themes) or match the border with the background color. But that comes with the drawback of the control having different sizes HC vs. non-HC or potentially having rendering issues (border + background usually doesn't look as nice as just plain background) so either you deal with some rendering weirdness or you have some adjustments in code.